inference-engine/src/mkldnn_plugin/mkldnn_node.h

```diff
@@ -361,7 +361,8 @@ class MKLDNNNode {
         return engine;
     }
 
-    // can be called after init optimal primitive descriptor
+    void initInPlace();
+
```

Comment:
Please move to the `private` section. But since it is used in only one place, it is better to inline this method.